File: Codeforces/A_236.cpp
// https://codeforces.com/contest/236/problem/A
// solved: 2013-08-25
#include <iostream>
#include <string>
using namespace std;
int main()
{
    int letters[26] = {0};
    string name;
    cin >> name;

    for (int i = 0; i < name.size(); i++)
    {
        letters[name[i] - 97]++;
    }
    int n = 0;
    for (int i = 0; i < 26; i++)
    {
        if (letters[i])
            n++;
    }
    if (n % 2 == 0)
        cout << "CHAT WITH HER!" << endl;
    else
        cout << "IGNORE HIM!" << endl;
    return 0;
}

File: Codeforces/A_236.java
import java.util.*;
public class A_236 {
	public static void main(String[] args){
		Scanner scan = new Scanner(System.in);
		String name = scan.nextLine();
		int distinctLett = 0;
		boolean[] letters = new boolean[100];
		Arrays.fill(letters, false);
		for(int i = 0; i < name.length(); i++){
			int index = Character.getNumericValue(name.charAt(i));
			if(!letters[index]){
				letters[index] = true;
				distinctLett ++;
			}
		}
		if(distinctLett % 2 == 0)
			System.out.println("CHAT WITH HER!");
		else
			System.out.println("IGNORE HIM!");
		scan.close();
		
	}
}